dotnet/sample-apps/aws-sdk/wrapper/SampleApps/AwsSdkSample/aws-lambda-tools-defaults.json

```diff
@@ -0,0 +1,23 @@
+{
+  "Information": [
+    "This file provides default values for the deployment wizard inside Visual Studio and the AWS Lambda commands added to the .NET Core CLI.",
+    "To learn more about the Lambda commands with the .NET Core CLI execute the following command at the command line in the project root directory.",
+    "dotnet lambda help",
+    "All the command line options for the Lambda command can be specified in this file."
+  ],
+  "profile": "default",
+  "region": "us-east-1",
+  "configuration": "Release",
+  "framework": "netcoreapp3.1",
+  "function-runtime": "dotnetcore3.1",
+  "function-memory-size": 256,
```

Comment:
Most of Lambda user is using 512 memory

Reply:
Removed this file as we're using terraform to deploy the Lambda, not aws lambda cli